### PR TITLE
feat: Add a GetPicutreCells function

### PR DESCRIPTION
**Pull request by yicixin**
# PR Details

## Description
When I import an xlsx, I need to go through all the cells to get the pictures,  each call to the getPicture function traverses all the pictures, why not just provide a way to get all the pictures at once.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sides-flow** [comment]
I could really use this functionality as well!
I hope it gets merged soon, thanks!

**Comment by yicixin** [comment]
When will this commit can be merged? 🤣

**Comment by xuri** [comment]
Thanks for your pull request. I need some time to review.

File: picture.go [hunk 1]
```diff
@@ -614,6 +614,52 @@ func (f *File) GetPictures(sheet, cell string) ([]Picture, error) {
...
+//	}
+//	for cellName, pics := range picsMap {
+//		for _, pic := range pics {
+//			name := fmt.Sprintf("image%d%s", idx+1, pic.Extension)
+//			if err := os.WriteFile(name, pic.File, 0644); err != nil {
+//				fmt.Println(err)
+//			}
+//		}
+//	}
+func (f *File) GetAllPictures(sheet string) (map[string][]Picture, error) {
```
Comment (xuri) [inline picture.go:644]: Every exported function should have unit tests and the line coverage should approach 100%.

**Review by xuri — CHANGES_REQUESTED** [review]
Sorry for the late reply. I've left some comments.

**Review by xuri — APPROVED** [review]
LGTM. Thanks for your contribution. I have made some changes based on your branch. This feature will be released in the next version.

**Pull request merged**